app/views/hub/faq_items/_header.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1 class="h2"><%= link_to("FAQ Categories", hub_faq_categories_path) + " < " + link_to(item.faq_category.slug, hub_faq_category_path(id: item.faq_category)) + " < #{item.slug} (##{item.id})" %></h1>
+<div class="grid">
```

Comment:
non-blocking nit: make this check if `item` is provided and if not, render like show it's shown in `new.html.erb` (without the `item` info + deletion)